src/commands/mod.rs

```diff
@@ -1435,14 +1435,22 @@ fn log_template(settings: &UserSettings) -> String {
         "styled" => format!("shortest({desired_id_len})"),
         _ => format!("short({desired_id_len})"),
     };
+    let author_template = match settings.log_author_format().as_str() {
+        "none" => None,
+        "full" => Some("author"),
+        "name" => Some("author.name()"),
+        "username" => Some("author.username()"),
+        _ => Some("author.email()"),
+    }
+    .map_or("".to_owned(), |t| t.to_owned() + ",");
     let default_template = format!(
         r#"
             label(if(current_working_copy, "working_copy"),
               separate(" ",
                 if(divergent,
                   label("divergent", change_id.{prefix_format} "??"),
                   change_id.{prefix_format}),
-                author.email(),
+                {author_template}
```

Comment:
I wish we could make the default template a single value without string interpolation, so the user can set it themselves. Maybe package managers (including internal ones at a company) would like to make some change in the template and still have the configs work, but that's not currently possible.

One option would be to have without `author` without any method called on it use the configured format. We could then have:
```
author: configured format
author.full(): Some One <some.one@example.com>
author.name(): Some One
author.email(): some.one@example.com
author.username(): some.one
```

For timestamps, we could have:
```
author.timestamp(): configured format
author.timestamp().ago(): 5 hours ago
author.timestamp().absolute(): 2023-02-11 10:30:14
```

It would also be nice to have a consistent way of specifying that default format.

Obviously, this is not a complaint about this PR.

Reply:
A version of #1190 could also be useful here -- if the default template were to be defined as modular "aliases", `{author} {timestamp}` etc., then those aliases could be redefined by the users without changing the default template itself (it's probably not all that different from what you're suggesting, I guess I'm just boosting #1190 :P).

Reply:
Yes, I was thinking that template aliases might be a good way to specify a default template. I hadn't thought about exactly how it would look, though. Maybe like this:
```
[default-templates]
signature(x) = x.username()
timestamp(x) = x.ago()
```
That's using the same syntax as in the hypothetical `[template-aliases]`, but specified in a separate table. Maybe there's a better way of doing it where we use the `[template-aliases]` table itself.